modules/UI.js

```diff
@@ -0,0 +1,35 @@
+import { awesomeBook, saveToLocalStorage } from './DBMsystem.js';
+
+/* eslint-disable */
+const removeBtnOnClick = (e) => {
+  e.preventDefault();
+  awesomeBook.removeBook(e.target.id);
+  displayBooklist(awesomeBook.books);
+  saveToLocalStorage();
+};
```

Comment:
- [ ] Please kindly do not disable linters, it's good to fix our linters errors it helps us to write clean and professional codebase 🙃 😉

Reply:
Thank you for your support.